src/main/java/com/kodilla/ecommercee/controller/user/UserController.java

```diff
@@ -1,26 +1,34 @@
 package com.kodilla.ecommercee.controller.user;
 
+import com.kodilla.ecommercee.controller.product.DBProductService;
+import com.kodilla.ecommercee.domain.user.UserBlockDto;
 import com.kodilla.ecommercee.domain.user.UserEntityDto;
+import com.kodilla.ecommercee.domain.user.UserLoginDto;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("/v1/user")
 public class UserController {
+    @Autowired
+    private UserMapper userMapper;
+    @Autowired
+    private DBUserService userService;
 
     @RequestMapping(method = RequestMethod.POST, value = "createUser", consumes = MediaType.APPLICATION_JSON_VALUE)
-    public void createUser(@RequestBody UserEntityDto userEntityDto){
+    public UserEntityDto createUser(@RequestBody UserEntityDto userEntityDto){
+        return userMapper.mapToUserEntityDto(userService.saveUser(userMapper.mapToUserEntity(userEntityDto)));
```

Comment:
Pomimo unique = true w loginie mogłem stworzyć kilku takich samych użytkowników 
![image](https://user-images.githubusercontent.com/34661411/72742378-e07cab80-3ba9-11ea-8311-f262bad60d97.png)
Sprawdź czy u ciebie też tak jest

Reply:
tak się da w momencie kiedy sprawdzasz w MySQL ale jak próbujesz np. zrobić w test suite dwa to też się da gdy testujes MySQL ale jeżeli robisz to na bazie H2 to już ona wywala błąd spróbuj i zobaczysz że w H2 działa, pyanie czy na zewnętrznym serwerze jak Heroku działa to dobrze ale to już do sprawdzenia przy testach

Reply:
Okej znalazłem chyba gdzie jest problem. Gdy spring tworzy tabele users, to nie zaznacza, że wartość jest unikalna 
![image](https://user-images.githubusercontent.com/34661411/72764814-daa6ba80-3be9-11ea-8417-f83d22c9762f.png)
Poszukałem trochę i znalazłem. Dla 
spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL**5**Dialect w application.propeties Spring nie tworzy wartości unikalnych. Dla
spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL**8**Dialect już tak
![image](https://user-images.githubusercontent.com/34661411/72765012-b26b8b80-3bea-11ea-9992-4d31457c1623.png)

Reply:
Ok dobra robot, ale zmieniać to już w properties czy jak zmargujemy controllery to zmienimy ? jeżeli na końcu to jak nie widzisz problemów to proszę o approve

Reply:
Można spytać Karola na rozmowie, czy to będzie miało jakieś większe konsekwencje i wtedy ustalimy

Reply:
jasne pozostawiam Tobie do rzucenia na spotkaniu z nienacka :)

Reply:
czyli on w żadnym wypadku nie pilnuje wartości unikalnych?